server/statistics/kind.go

```diff
@@ -101,15 +101,12 @@ func (k StoreStatKind) String() string {
 type sourceKind int
 
 const (
-	direct  sourceKind = iota // there is a corresponding peer in this store.
-	inherit                   // there is no a corresponding peer in this store and there is a peer just deleted.
-	adopt                     // there is no corresponding peer in this store and there is no peer just deleted, we need to copy from other stores.
+	inherit sourceKind = iota // there is a corresponding peer in this store.
```

Comment:
Does use `direct, inherit` more suitable? change adopt to inherit and remove old inherit.

Reply:
Good idea